guides/posts/api-management-practices/post.md

```diff
@@ -6,7 +6,7 @@ publishedDate: 2021-10-26T10:25:53.100Z
 lastModifiedDate: 2021-10-26T10:25:53.100Z
 authors:
     - saad
-category: api
+category: Best Practices
```

Comment:
You need to add the new categories [here](https://github.com/RapidAPI/DevRel-Stack-Data/blob/production/categories/data.json) as well. You can use any color that you would like.